graphql/resolvers.go

```diff
@@ -1569,15 +1569,24 @@ func (r *queryResolver) PatchTasks(ctx context.Context, patchID string, sorts []
 			taskSorts = append(taskSorts, task.TasksSortOrder{Key: key, Order: order})
 		}
 	}
+
+	v, _ := r.sc.FindVersionById(patchID)
```

Comment:
This query could fail for unscheduled patches. We never pass in unscheduled patchIds into this resolver from the UI so it may be useful to check the error here.

Reply:
I was able to remove this line all together. The original intent was conditionally query generated tasks based on version properties but that's no longer necessary since we will always fetch unactivated and unscheduled tasks from the Schedule Tasks modal